docs/src/administration/users.md

```diff
@@ -72,67 +60,27 @@ Available commands:
 * `platform user:role`
   * View or change a user's role
 
-For example, the following command would add `alice@example.com` with **Project Admin** role in the current project.
+ **Example**
+
+ Add `user1@example.com` to the project with Project Admin permissions:
```

Comment:
This is the same task as above, so I'd just move it there (with the tabs, as noted above).